blueprints/grafana/creator.libsonnet

```diff
@@ -35,6 +36,20 @@ function(policyFile, cfg) {
     for component in flowControlComponents
   ])),
 
+  local infraMeters =
+    if std.objectHas(policyJSON.spec.resources, 'infra_meters')
+    then policyJSON.spec.resources.infra_meters
+    else {},
+
+  local receiverName = std.objectFields(infraMeters)[0],
```

Comment:
@karansohi 
Need to loop the infra meters, it's an array! Also, accessing by index can crash if it's not present